appinventor/components/src/com/google/appinventor/components/scripts/ComponentProcessor.java

```diff
@@ -2229,7 +2229,7 @@ private static String activityElementToString(ActivityElement element)
       throws IllegalAccessException, InvocationTargetException {
     // First, we build the <activity> element's opening tag including any
     // receiver element attributes.
-    StringBuilder elementString = new StringBuilder("    <activity ");
+    StringBuilder elementString = new StringBuilder("    <activity android:exported=\\\"true\\\" ");
```

Comment:
I'm not sure this is the correct behavior. In particular, the `@ActivityElement` does provide a `exported` field, which defaults to the empty strong (so it's not written). If someone explicitly sets that field, I think this causes issues in the XML parsing (but I haven't tested, so maybe not). We might want to adjust the annotation definition to have the default be "true" rather than "", or maybe even "false" so that it's more conservative, i.e., making a public activity requires taking action on the developer's part.